hail/src/main/scala/is/hail/annotations/RegionMemory.scala

```diff
@@ -58,9 +58,9 @@ final class RegionMemory(pool: RegionPool) extends AutoCloseable {
 
   private def allocateBigChunk(size: Long): Long = {
     val o = pool.getChunk(size)
```

Comment:
To make this clearer, do something like `val (addr, bytesAllocated) = pool.getChunk(size)`